isofit/radiative_transfer/radiative_transfer.py

```diff
@@ -221,10 +209,10 @@ def calc_rdn(self, x_RT, rfl_dir, rfl_dif, Ls, geom):
         ret = (
             L_atm
             + (
-                self.E_coupled[0] * rfl_dir  # bidirectional flux
-                + self.E_coupled[1] * rfl_dif  # hemispherical-directional flux
-                + self.E_coupled[2] * bg_dir  # directional-hemispherical flux
-                + self.E_coupled[3] * bg_dif  # bi-hemispherical flux
+                E_bi_direct * rfl_dir  # bi-directional flux
+                + E_hemi_direct * rfl_dif  # hemispherical-directional flux
+                + E_direct_hemi * bg_dir  # directional-hemispherical flux
+                + E_bi_hemi * bg_dif  # bi-hemispherical flux
```

Comment:
@pgbrodrick @davidraythompson Here's an updated nomenclature for the coupled terms in our new radiance model. Let me know what you think!

Reply:
Nice! My only nit pick is to ask why we're prefixing with E instead of L if all these quantities are radiances -   typically E is reserved for irradiances (as in Schaepman Strub 2006).  Otherwise great.

Reply:
Good point! Updated with commit [03fa11f](https://github.com/isofit/isofit/pull/524/commits/03fa11fd75101172459d416136b13a1e29a6a710).